### ISSUE TITLE
Element inside Modal unable to be tab through in safari browser

### ISSUE THREAD

**Issue by huangxingyang** [OfficeDev/office-ui-fabric-react#11215]
### Environment Information

- **Package version(s)**: 7.55.3
- **Browser and OS versions**: Safari 12.1.2 Mac 10.14.6

### Please provide a reproduction of the bug in a codepen: 
[Example Here](https://codepen.io/xingyang-huang/pen/BaaqEYR?&editable=true)
Please try to running it in Safari browser.

#### Actual behavior:
1. click open model
2. after the model is showing up, press the tab key
3. the tab key stays in the first button
#### Expected behavior:
The tab key should walk through all the buttons.
### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: High

**Comment by kenotron** [comment]
I don't think this is directly related to modal. I took the content out of the modal and try to tab around, and Safari just acts like it is flipping between the page and the content:

https://cdpn.io/kenotron/debug/MWWPMBo/NQkzYBBYRVoA

In fact, I have removed EVERYTHING and still can't focus on the button:

https://codepen.io/kenotron/pen/BaaqgGN
(debug mode) https://cdpn.io/kenotron/debug/BaaqgGN/XxMVwjjwXPBr

It just seems like Safari isn't a very accessible browser BY DEFAULT. Here's a recording of me trying to focus on anything inside the page

![Screen Recording 2019-11-14 at 3 31 47 PM mov](https://user-images.githubusercontent.com/34725/68904962-49e8c580-06f4-11ea-8bbe-1b30ac38be1e.gif)

Apparently you have to turn on accessibility mode in Safari to make this work:

![image](https://user-images.githubusercontent.com/34725/68905084-d0050c00-06f4-11ea-8f6d-caae27a19828.png)

**Comment by kenotron** [comment]
After applying that setting, the tab navigation works in your example.

**Comment by huangxingyang** [comment]
Thanks @kenotron !!
